app.js

```diff
@@ -39,6 +39,12 @@ app.post(
   "/caniuse/update",
   bodyParser.json({ verify: rawBodyParser }),
   require("./routes/caniuse/update").route
+
+app.options("/github/:org/:repo/contributors", cors({ methods: ["GET"] }));
```

Comment:
can shorten it to `/gh/:org/:repo/contributors` if too long

Reply:
It’s fine